client/my-sites/stats/stats-tabs/index.jsx

```diff
@@ -31,7 +31,8 @@ class StatsTabs extends Component {
 		} else {
 			data?.map( ( day ) =>
 				tabs.map( ( tab ) => {
-					if ( isFinite( day[ tab.attr ] ) ) {
+					// Exclude non-numeric values, e.g., NULL from unsupported stats fields.
+					if ( Number.isFinite( day[ tab.attr ] ) ) {
```

Comment:
```
isFinite( null ) === true
Number.isFinite( null ) === false
```

Reply:
Hmmmm that's why we love JavaScript right? 🥲